scripts/enforce-call-super-for-overrided-onupdate

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+function is_call_super_found {
+  # If class directly extends BaseProjectKeyAvailabilityAssuranceAction then call super.onUpdate is no needed
+  grep "extends BaseProjectKeyAvailabilityAssuranceAction" "$0" > /dev/null
```

Comment:
(regardless of the comments below) doesn't `$0` resolve to shell name?

```
$ function foo() { echo dollar0=$0; }
$ foo hello
dollar0=bash
```

Reply:
In this case it wasn't (I also don't know why)